deployment/overlays/pls/deployment.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: central-viewer
+spec:
+  selector:
+    matchLabels:
+      app: central-viewer
+  template:
+    spec:
+      containers:
+      - image: acrsensrnetregistry.azurecr.io/sensrnet/central-viewer:latest
```

Comment:
`latest` is voor nu OK ... maar zal richting officiëlere status naar fixed versions moeten wijzigen